testutil/x/counter/README.md

```diff
@@ -5,3 +5,64 @@ sidebar_position: 1
 # `x/counter`
 
 Counter is a module used for testing purposes within the Cosmos SDK.
+
+## Hooks
```

Comment:
Nice, but I meant creating a page at build-modules/hooks on the website instead. No one will ever find this otherwise :D

Basically, if you could move that section under docs/build-modules that would be great.
We should have as well an intro about what is hook, and link those pages: https://docs.cosmos.network/main/build/modules/epochs#hooks, https://docs.cosmos.network/main/build/modules/distribution#hooks

We can use code snippet, as `go reference` to link the counter module changes you've done here and example what everything does.